shell/common/shell.cc

```diff
@@ -987,6 +988,7 @@ void Shell::OnAnimatorDraw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline,
             waiting_for_first_frame_condition.notify_all();
           }
         }
+        FML_DLOG(ERROR) << "Scene Size: " << external_size_bytes;
```

Comment:
This is around where we'd want to call `Dart_SomethingLikeNotifyGCWeightChangedOrWhatever`, so it could realize that doing an unexpected GC here may radically reduce memory usage (i.e. because the Scene contained a large image but doesn't anymore).